packages/relay/src/lib/services/ethService/blockService/BlockService.ts

```diff
@@ -387,7 +387,7 @@ export class BlockService implements IBlockService {
           s: constants.ZERO_HEX,
           to: log.address,
           transactionIndex: log.transactionIndex,
-          type: constants.TWO_HEX, // 0x0 for legacy transactions, 0x1 for access list types, 0x2 for dynamic fees.
+          type: constants.ZERO_HEX, // 0x0 for legacy transactions, 0x1 for access list types, 0x2 for dynamic fees.
```

Comment:
nit: edit the comment to `// 0x0 for legacy and synthetic transactions, 0x1 for access list types, 0x2 for dynamic fees.`